pkg/sql/show_tenant.go

```diff
@@ -50,10 +104,48 @@ func (n *showTenantNode) Next(_ runParams) (bool, error) {
 	return true, nil
 }
 func (n *showTenantNode) Values() tree.Datums {
+	tenantId := tree.NewDInt(tree.DInt(n.info.ID))
+	tenantName := tree.NewDString(string(n.info.Name))
+	tenantStatus := tree.NewDString(n.info.State.String())
+	if !n.withReplication {
+		// This is a simple 'SHOW TENANT name'.
+		return tree.Datums{
+			tenantId,
+			tenantName,
+			tenantStatus,
+		}
+	}
+
+	// This is a 'SHOW TENANT name WITH REPLICATION STATUS' command.
+	sourceTenantName := tree.DNull
+	sourceClusterUri := tree.DNull
+	replicationJobId := tree.NewDInt(tree.DInt(n.info.TenantReplicationJobID))
+	replicatedTimestamp := tree.DNull
+	retainedTimestamp := tree.DNull
+
+	// New replication clusters don't have replicationInfo initially.
```

Comment:
dropped, it's not clear and repeating the comment above. See the comment and todo after calling `mgr.GetStreamIngestionStats()` above? hopefully this would make sense.